endpoints/cps.yaml

```diff
@@ -283,3 +416,77 @@ components:
           $ref: "../components/Position.yaml"
         orientation:
           $ref: "#/components/schemas/Gravity"
+    InitializeResult:
```

Comment:
The initialize result is slightly different based on a successful result or not. For example we also return `"fantasmo_unavailable_reason": "No parking in radius"` when hitting this endpoint and coordinates are out of range